doc/ADMIN.md

```diff
@@ -2,12 +2,55 @@
 
 You can run Nextcloud commands from the command line using:
 
-```
+```bash
 sudo -u __APP__ php__PHPVERSION__ --define apc.enable_cli=1 __INSTALL_DIR__/occ ...
 ```
 
 Alternatively, you may open a 'Nextcloud shell' with `sudo yunohost app shell __APP__`, then run `php occ ...`
 
+### Change data folder location
```

Comment:
Related to https://github.com/YunoHost/doc/blob/master/pages/04.applications/10.docs/nextcloud/app_nextcloud.md#ii-migrate-nextcloud-data-to-a-larger-partition which is actually supposed to be removed and/or migrated to the app's ADMIN.md as a consequence of https://github.com/YunoHost/doc/pull/2417

This old piece of doc actually seems to recommend a savage cp + edition of the config.php + the data dir setting

The symlink solution seems better but I don't remember why exactly but this ultimately also causes issues later on (maybe with backups if i remember correctly)

There is also https://yunohost.org/en/administer/tutorials/external_storage which to me sounds like the most reliable solution and uses bind mounts

Ideally we should harmonize all this because this is quite a reccuring question/issue and currently there's no way to move apps data in a simple yet long-term robust way